_posts/2019-05-16-released-v6.md

```diff
@@ -0,0 +1,76 @@
+  The effort is still on going at [the tracking issue](https://github.com/chainer/chainer/issues/6423) with [the spreadsheet of op-wise implementation status](https://docs.google.com/spreadsheets/d/1B4E78tw9Awgpcdn5G7zsQ8NVFYJdOoJlIQg42QxKNfU).
+  We continue expanding the op coverage towards next v7 release.
+  Contributions are always welcomed!
+- **More Function coverage**.
+  Most users will start using ChainerX through Chainer's existing interface (just by replacing NumPy/CuPy array with ChainerX array).
+  When ChainerX does not have an implementation for an operation, Chainer automatically falls back to NumPy/CuPy-based implementation.
+  It basically works without any fix for most functions, but sometimes not.
+  We are fixing such bugs to enlarge the coverage of functions for ChainerX usage.
+  The effort is accompanied with the introduction of new simplistic test fixtures for function tests (you can find [the tracking issue](https://github.com/chainer/chainer/issues/6071)).
+  The current test coverage is over 40%.
+  We hope the coverage will reach 100% in v7.
+  Contributions are always welcomed here, too!
+- **More example coverage**.
+  Most examples now support ChainerX.
+  By specifying ChainerX's device names (e.g. `native` for CPU and `cuda:0`, `cuda:1`, ... for GPU), examples run with ChainerX arrays.
+  It also means that the coverage of ChainerX support in Chainer's features in general is expanding.
+
+You can find the [previous blog post](https://chainer.org/announcement/2018/12/03/chainerx.html) for its background and overview,
+ and [ChainerX Documentation](https://docs.chainer.org/en/v6.0.0/chainerx/index.html) for the installation guide, tutorial, and reference.
+
+
+## Other updates
+
+This release also includes many features other than ChainerX.
+We list up notable updates as follows.
+
+- **More mixed precision support**.
+  Chainer v6 introduces _mixed precision mode_ and _dynamic loss scaling_ for better support of mixed precision training.
+  Mixed precision mode is enabled by setting `CHAINER_DTYPE=mixed16` or `chainer.global_config.dtype = chainer.mixed16`.
+  In this mode, Chainer automatically chooses either of `float16` or `float32` which is appropriate for performance and precision balance.
+  Dynamic loss scaling, originated from [Apex](https://github.com/NVIDIA/apex), automatically adjusts the scaling coefficient of backprop to avoid underflow.
+- **Device API**.
+  We introduce a new device API for better interoperability between backends (including ChainerX).
+  It unifies the way to specify a device and transfer data between devices.
+  In particular, unified device specifier is introduced.
+  It is based on ChainerX's device specifier of format `'backend:id'`, e.g. `'native:0'` and `'cuda:N'` (where `N` is the CUDA device id).
+  For native (CPU), the id part can be omitted (like `'native'`).
+  For conventional devices backed by NumPy-like modules, the name is like `@numpy`, `@cupy:N`, and `@intel64`.
```

Comment:
Since "like" is also used in the sentence above, and it's a bit informal.
```suggestion
  For conventional devices backed by NumPy-like modules, the name is `@numpy`, `@cupy:N`, and `@intel64`.
```